packages/harpguru-core/src/components/note-display-fragment/note-display-fragment.tsx

```diff
@@ -0,0 +1,89 @@
+import { useGlobal } from 'reactn'
+import { View, Text, StyleSheet } from 'react-native'
+import React from 'react'
+
+import { ExperienceModes } from '../../types'
+import { getSizes, colors } from '../../styles'
+
+type NoteTuple = [string, string] | [string, undefined] | [undefined, undefined]
+
+type NoteDisplayFragment = {
+  readonly displayValue: ReadonlyArray<NoteTuple>
```

Comment:
This and NoteTuple are duplicates of what will be exported from the `getRenderableToneTuples` function. Perhaps they shouldn't be exported from there in fact and should really be part of the root types directory.